src/chainparams.cpp

```diff
@@ -398,13 +398,21 @@ class CMainParams : public CChainParams {
 
         for (const auto& str : lelantus::lelantus_blacklist) {
             GroupElement coin;
-            coin.deserialize(ParseHex(str).data());
+            try {
+                coin.deserialize(ParseHex(str).data());
+            } catch (...) {
+                continue;
+            }
             consensus.lelantusBlacklist.insert(coin);
         }
 
         for (const auto& str : sigma::sigma_blacklist) {
             GroupElement coin;
-            coin.deserialize(ParseHex(str).data());
+            try {
+                coin.deserialize(ParseHex(str).data());
+            } catch (...) {
+                continue;
```

Comment:
Is this `catch` reached given the current chain?

Reply:
the same